src/Polytopes/Cone/properties.jl

```diff
@@ -160,12 +142,9 @@ true
 ispointed(C::Cone) = pm_cone(C).POINTED
 
 """
-    isfulldimensional(C)
+    isfulldimensional(C::Cone)
 
-Determine whether the cone is full dimensional
-
-# Arguments
-- `C::Cone`: A cone.
+Determine whether `C` is full dimensional.
```

Comment:
Elsewhere you write this as "full-dimensional" ?
```suggestion
Determine whether `C` is full-dimensional.
```